Mathlib/MeasureTheory/Integral/IntegralEqImproper.lean

```diff
@@ -859,6 +859,15 @@ theorem _root_.HasCompactSupport.integral_Iic_deriv_eq (hf : ContDiff ℝ 1 f)
   rw [hasCompactSupport_iff_eventuallyEq, Filter.coclosedCompact_eq_cocompact] at h2f
   exact h2f.filter_mono _root_.atBot_le_cocompact |>.tendsto
 
+theorem integral_of_hasDerivAt_of_tendsto
```

Comment:
Could you move it to another section like `univFTC`, that we will maybe extend later on? This lemma doesn't really belong to `IicFTC`.